annotations/static/annotations/js/annotators/text.js

```diff
@@ -129,6 +129,9 @@ ConceptCreator = {
                             id="concept-creator-description"
                             v-model="description">
                         </textarea>
+                        <span class="help-block text-warning" v-if="showDescriptionWarning()">
+                            Description must be at least 3 words or 10 characters long.
+                        </span>
```

Comment:
This text should always be shown. Otherwise if the user doesn't enter anything in the description field, they don't see the error message.